test/gtest/common/test_helpers.cc

```diff
@@ -471,6 +471,31 @@ bool is_rdmacm_netdev(const char *ifa_name) {
     return strstr(guid_buf, "0000:0000:0000:0000") == NULL;
 }
 
+bool is_sockcm_netdev(const char *ifa_name) {
+    char file_name[PATH_MAX];
+    int ch;
+    FILE *fp;
+
+    /*
+     * check if the device is an ethernet device
```

Comment:
why do we care if it's ethernet
TCP could work over any type of L2. for example: IPoIB

Reply:
so lets remove this function at all
assume sockcm works on all net devices